tests/features.py

```diff
@@ -0,0 +1,41 @@
+from flask import current_app
+from mock import patch
+
+
+def enable_feature(name, value=True):
+    PATCH = {'FEATURE_FLAGS_{}'.format(name.upper()): value}
+
+    def decorator(f):
+        if isinstance(f, type):
+            class Proxy(object):
+                def __init__(self, *args, **kwargs):
+                    self._instance = f(*args, **kwargs)
+
+                def __getattribute__(self, name):
+                    try:
+                        attr = super(Proxy, self).__getattribute__(name)
+                    except AttributeError:
+                        pass
+                    else:
+                        return attr
+
+                    attr = self._instance.__getattribute__(name)
+
+                    if name.startswith('test_') and callable(attr):
+                        return decorator(attr)
+                    else:
+                        return attr
+
+            return Proxy
+
+        def wrap(*args, **kwargs):
+            with patch.dict(current_app.config, PATCH):
+                return f(*args, **kwargs)
+
+        return wrap
+
+    return decorator
+
+
+def disable_feature(name):
+    return enable_feature(name, False)
```

Comment:
Has this gotten the go-ahead?  There was a discussion last time in #dm-development about overriding the config variable if possible.  Did that idea get torpedoed?